pages/about.html

```diff
@@ -0,0 +1,126 @@
+            <P class="primary-text"> Projecto Muda is a movement that started back in 2018 and it aims to raise young people awareness on social problems. Project Muda believes that anyone can help and you do not need too much to do so. This movement has been able to help many families and the 2022 Edition won't be any different. </P>
+            <div class="meeting-date-place">
+                <h2 class="date"> Please contact us per Email for any further questions about CC Global Summit 2022 </h2>
+                <a href="#" class="place"> projectomuda@gmail.com</a>
+            </div>
+        </div>
+    </section>
+    <section class="logo-info">
+        <div class="logo-info-title">
+            <h2> Projecto Muda Logo </h2>
+            <div class="separator separator-logo-info"></div>
+        </div>
+        <div class="logo-info-description">
+            The logo of Projecto Muda was decided <span> by the founders of the movement in 2018.
+            </span>
+        </div>
+        <div class="logo-info-img">
+            <img src="../img/logo_muda.jpg" alt="">
+        </div>
+    </section>
+    <section class="past-editions">
+        <div class="logo-info-title past-editions-title">
+            <h2> See past Projecto Muda Editions </h2>
+            <div class="separator separator-logo-info"></div>
+        </div>
+        <div class="past-editions-description logo-info-description">
+            Take a look at the last two Project Muda Editions which <span>  took place in Maputo </span>
+        </div>
+        <div class="past-editions-img-wrapper">
+            <div class="past-editions-img past-editions-img-1">
+                <p> 2019 </p> 
+                <span> Children day </span>
+            </div>
+            <div class="past-editions-img past-editions-img-2">
+                <p> 2019 </p> 
+                <span> Solidary Christmas </span>
+            </div>
+        </div>
+    </section>
+    <section class="footer">
+        <div class="partners">
+            <div class="logo-info-title partners-title">
+                <h2> Partner </h2>
+                <div class="separator separator-logo-info"></div>
+            </div>
+            <div class="partner-logo">
+                <p class="partner-logo-p partner-logo-p-1"> Google </p>
+                <p class="partner-logo-p partner-logo-p-2"> mozilla </p>
+                <p class="partner-logo-p partner-logo-p-3"> NAVER </p>
+                <p class="partner-logo-p partner-logo-p-4"> Pepsi </p>
+                <p class="partner-logo-p partner-logo-p-5"> EJ Media </p>
+            </div>
+        </div>
+        <div class="rights">
+            <div class="company">
+                @Projecto Muda
+            </div>
+            <div class="rights-reserved">
+                2022 Projecto Muda. <span> All rights reserved</span>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] For making your project use semantic tags correctly and the HTML markup is well structured and follows a clear hierarchy, I recommend you to use for partner section `section` tag and for footer use `footer` tag and make sure that footer contains a logo aligned to the left and text aligned to the right.

Your About footer page look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/197205539-827b4572-1aac-434c-b513-d636224519ab.PNG)

The Design About page for footer section Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/197205714-d215ca96-9837-4f20-8f18-33fb565942ba.PNG)